src/govuk/components/notification-banner/_index.scss

```
@@ -0,0 +1,87 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  /* stylelint-disable */
+
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: 4px solid $govuk-focus-colour;
```

I think this should be

```suggestion
      outline: $govuk-focus-width solid $govuk-focus-colour;
```

Although `$govuk-focus-width` is 3px… but that's what we're using on the error summary.